s3_scrubber/Cargo.toml

```diff
@@ -30,10 +34,8 @@ aws-config = { workspace = true, default-features = false, features = ["rustls",
 
 pageserver = {path="../pageserver"}
 
-
 tracing.workspace = true
 tracing-subscriber.workspace = true
 clap.workspace = true
-
-atty = "0.2"
-tracing-appender = "0.2"
\ No newline at end of file
+tracing-appender = "0.2"
+histogram = "0.7"
```

Comment:
Could you add this newline to either one of your PRs?

Reply:
A newline went into https://github.com/neondatabase/neon/pull/5171